tests/plugins/test_sql.py

```diff
@@ -13,7 +13,7 @@ def test_sql_parser(database_url):
     with Resource(database_url, dialect=dialect) as resource:
         assert resource.schema == {
             "fields": [
-                {"name": "id", "type": "integer"},
+                {"name": "id", "type": "integer", "constraints": {"required": True}},
```

Comment:
It's not needed. Please see the review summary comment

Reply:
Should be fixed now, indeed had sqlalchemy 1.3 for some reason.